### PR TITLE
Screenshots automated via fastlane

### PR DESCRIPTION
**Pull request by FesenkoG**

### CODE REVIEW

**Comment by skywinder** [comment]
@FesenkoG We have to separate Screenshot Test scheme from normal tests. Since it's failing with not initiated wallet

File: DiveLaneUITests/DiveLaneUITests.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        snapshot("Wallets")
+        let tablesQuery = app.tables
+        tablesQuery/*@START_MENU_TOKEN@*/.buttons["Info"]/*[[".cells.buttons[\"Info\"]",".buttons[\"Info\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        snapshot("WalletInfo")
+        app.navigationBars["Wallet"].buttons["Wallets"].tap()
+        walletsNavigationBar.buttons["Wallets"].tap()
+        tablesQuery.buttons["+"].tap()
+        tablesQuery.searchFields["Search"].tap()
+        snapshot("TokenSearch")
+        tablesQuery/*@START_MENU_TOKEN@*/.staticTexts["0x1ca43a170bad619322e6f54d46b57e504db663aa"]/*[[".cells.staticTexts[\"0x1ca43a170bad619322e6f54d46b57e504db663aa\"]",".staticTexts[\"0x1ca43a170bad619322e6f54d46b57e504db663aa\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
```
Comment (skywinder) [inline DiveLaneUITests/DiveLaneUITests.swift]: Don't stick to the hardcoded labels, that could be different on different machines

File: DiveLaneUITests/DiveLaneUITests.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        setupSnapshot(app)
+        app.launch()
+        snapshot("MainScreen")
+        let walletsNavigationBar = app.navigationBars["Wallets"]
+        walletsNavigationBar.buttons["Add"].tap()
+        snapshot("Wallets")
+        let tablesQuery = app.tables
+        tablesQuery/*@START_MENU_TOKEN@*/.buttons["Info"]/*[[".cells.buttons[\"Info\"]",".buttons[\"Info\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        snapshot("WalletInfo")
+        app.navigationBars["Wallet"].buttons["Wallets"].tap()
```
Comment (skywinder) [inline DiveLaneUITests/DiveLaneUITests.swift]: Be careful with such texts. Remember about localisations. SO it's better request them by index for example

File: DiveLaneUITests/DiveLaneUITests.swift (hunk 1 above)
Comment (FesenkoG) [reply DiveLaneUITests/DiveLaneUITests.swift]: Yes, this part of code was not supposed to be ready. I don't know how the design will be changed, so I made the easiest and fastest solution just to test that screenshots mechanism was working.

File: DiveLaneUITests/DiveLaneUITests.swift (hunk 2 above)
Comment (FesenkoG) [reply DiveLaneUITests/DiveLaneUITests.swift]: Don't accessibility identifiers solve problems with localisation?

File: DiveLaneUITests/DiveLaneUITests.swift (hunk 1 above)
Comment (skywinder) [reply DiveLaneUITests/DiveLaneUITests.swift]: accessibility id's can do the trick 👍

**Pull request merged**